llvm/utils/UpdateTestChecks/asm.py

```diff
@@ -32,7 +32,7 @@ class string:
 )
 
 ASM_FUNCTION_ARM_RE = re.compile(
-    r"^(?P<func>[0-9a-zA-Z_$]+):\n"  # f: (name of function)
+    r'^(?P<func>[0-9a-zA-Z_$]+):[ \t]*@+[ \t]*@"?(?P=func)"?\n'  # f: (name of function)
```

Comment:
LGTM. Seems that other targets already check the comment that contains the function name